src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -19,17 +19,19 @@
 import logging
 from typing import List, Optional, Union
 
+
+try:
+    from typing import Unpack
+except ImportError:
+    pass
```

Comment:
```suggestion
if sys.version_info >= (3, 11):
    from typing import Unpack
else:
    from typing_extensions import Unpack
```
This will make the CI happier :)

Reply:
Thanks. Didn't know about sys.version_info.